content/en/developers/dogstatsd/unix_socket.md

```diff
@@ -39,8 +39,17 @@ To enable the Agent DogStatsD UDS:
 {{< tabs >}}
 {{% tab "Host" %}}
 
-1. Edit the [Agent's main configuration file][1] to set `dogstatsd_socket` to the path where DogStatsD should create its listening socket:
-
+1. Create a socket file for DogStatsD to use as a listening socket. For example:
+   ```shell
+   sudo mkdir -p /var/run/datadog/
+   ```
+1. Ensure that the `dd-agent` user has read and write permissions to the socket file:
+   ```shell
+   sudo chown dd-agent:dd-agent /var/run/datadog/
+   ```
+1. Edit the [Agent's main configuration file][1]:
+   1. Set `use_dogstatsd` to `true`.
```

Comment:
```suggestion
   1. Set `use_dogstatsd` to `true`.
```
minor nit my eyes got a little confused seeing the 1. and 2. steps here and then below step 4 to restart your Agent. I think it's ok since it's indented, just an observation :)

Reply:
Yeah, you're right. I did this all in the web UI cos I was too lazy to create a branch and this is where it got me.

Reply:
This should be fixed now. These indented steps are sub-steps. They should look okay in the preview